src/main/java/de/espend/idea/php/annotation/doctrine/intention/DoctrineOrmRepositoryIntention.java

```diff
@@ -171,15 +205,32 @@ private void insertAttribute(@NotNull Editor editor, @NotNull PhpClass phpClass,
         }
     }
 
+    @Nls(capitalization = Nls.Capitalization.Sentence)
+    @NotNull
+    @Override
+    public String getName()
+    {
+        return getText();
+    }
+
     @NotNull
     @Override
     public String getFamilyName() {
         return "PhpAnnotations";
     }
 
+    @Override
+    public void applyFix(@NotNull Project project, @NotNull ProblemDescriptor descriptor)
+    {
+        if (descriptor.getPsiElement() == null) return;
+        if (descriptor.getPsiElement().getContainingFile() == null) return;
+
+        invoke(project, null, descriptor.getPsiElement().getContainingFile());
+    }
+
     @NotNull
     @Override
     public String getText() {
-        return "Add Doctrine Repository";
+        return "Add doctrine repository";
```

Comment:
Tests are failing because of this rename

Reply:
Fixed 👍 😄